.github/workflows/performance-tests.yml

```diff
@@ -36,14 +36,12 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # https://github.com/microsoft/vscode-ripgrep/issues/9
 
       - name: Performance (browser)
-        uses: GabrielBB/xvfb-action@v1
-        with:
-          run: yarn performance:startup:browser
+        shell: bash
+        run: yarn performance:startup:browser
 
       - name: Performance (Electron)
-        uses: GabrielBB/xvfb-action@v1
-        with:
-          run: yarn performance:startup:electron
+        shell: bash
```

Comment:
@xai Do you think omitting xvfb will be a problem for Electron? If yes, we'd need to switch to https://github.com/coactions/setup-xvfb as https://github.com/marketplace/actions/gabrielbb-xvfb-action is deprecated. Thanks!

Reply:
We need xvfb because we cannot run Electron headless. So yes, we should switch to the maintained action.

Reply:
Ok, no problem! I updated this PR. Can you please test if this works now? Thank you!

Reply:
I think we need to test by running the workflow, right?